packages/anoncreds/src/models/registry.ts

```diff
@@ -23,6 +23,7 @@ export interface AnonCredsRevocationRegistryDefinition {
   credDefId: string
   tag: string
   value: {
+    issuanceType: 'ISSUANCE_BY_DEFAULT' | 'ISSUANCE_ON_DEMAND'
```

Comment:
Revocation registry definition doesn't have `issuanceType` anymore. This is something specific to indy.

```suggestion
```